README.md

```diff
@@ -1 +1,9 @@
-# live_cell_timelapse_apoptosis_analysis
\ No newline at end of file
+# Live Cell Timelapse Apoptosis Analysis
+
+This repository contains the code to analyze live cell timelapse microscopy data of apoptosis in HeLa cells.
+The goal of this analysis repository is to build a pipeline and framework to analyze live cell timelapse microscopy data modes.
+Specifically, multi-channel fluorescence microscopy data.
+
+The sample data and upstream analysis of such data can be found in the sibling repository [live_cell_timelapse_apoptosis](https://github.com/WayScience/live_cell_timelapse_apoptosis).
```

Comment:
I would recommend changing the name of the repo to include image profiling like we did with pyroptosis to make it much more explicit.

Also, I would recommend using image analysis and image-based profiling in this sentence to be specific about what this sibling repo does.

```suggestion
The sample data and upstream analysis of such data, including image analysis with CellProfiler and image-profiling with CytoTable and pycytominer, can be found in the sibling repository [live_cell_timelapse_apoptosis](https://github.com/WayScience/live_cell_timelapse_apoptosis).
```